README.md

```diff
@@ -26,5 +26,16 @@ Repository for TDR transfer code
 
 * Start the application using `sbt run`
 
+* If css is not displaying on localhost:9000/dashboard run `npm run build`
```

Comment:
I think it's fine to skip the "If css is not displaying". People will have to run this regularly anyway. :slightly_smiling_face:

Reply:
Removed wording 👌